spec/Section 3 -- Type System.md

```diff
@@ -1489,8 +1489,9 @@ Expected Type | Provided Value   | Coerced Value
 `[Int]`       | `[1, "b", true]` | Error: Incorrect item value
 `[Int]`       | `1`              | `[1]`
 `[Int]`       | `null`           | `null`
-`[[Int]]`     | `[[1], [2, 3]]`  | `[[1], [2, 3]`
-`[[Int]]`     | `[1, 2, 3]`      | Error: Incorrect item value
+`[[Int]]`     | `[[1], [2, 3]]`  | `[[1], [2, 3]]`
+`[[Int]]`     | `[1, 2, 3]`      | `[[1], [2], [3]]`
+`[[Int]]`     | `[1, null, 3]`   | `[[1], null, [3]]`
```

Comment:
Both above cases are correct as is since this coercion applied only if:
http://facebook.github.io/graphql/June2018/#sec-Type-System.List
> If the value passed as an input to a list type is NOT A LIST